core/core-configuration-layer.el

```diff
@@ -689,7 +692,10 @@ path."
               (configuration-layer//get-latest-package-version-string
                pkg-name)))
       ;; (message "%s: %s > %s ?" pkg-name cur-version new-version)
-      (version< cur-version new-version))))
+      (if new-version
+          (version< cur-version new-version)
+        (cl-pushnew pkg-name configuration-layer--skipped-packages :test #'eq)
```

Comment:
See [question on SE](http://emacs.stackexchange.com/questions/7389/whats-the-difference-between-push-and-add-to-list). Specifically, there is a quote there from the doc for `add-to-list`: 
> This is handy to add some elements to configuration variables,
> but please do not abuse it in Elisp code, where you are usually
> better off using \`push' or \`cl-pushnew'.